src/EnergyPlus/SZVAVModel.cc

```diff
@@ -277,7 +277,7 @@ namespace SZVAVModel {
 
             if (HeatingLoad) { // Function UnitarySystems::calcUnitarySystemToLoad, 4th and 5th arguments are CoolPLR and HeatPLR
                 // set the water flow ratio so water coil gets proper flow
-                SZVAVModel.HeatCoilWaterFlowRatio = maxCoilFluidFlow / SZVAVModel.MaxHeatCoilFluidFlow;
+                if (SZVAVModel.MaxHeatCoilFluidFlow > 0.0) SZVAVModel.HeatCoilWaterFlowRatio = maxCoilFluidFlow / SZVAVModel.MaxHeatCoilFluidFlow;
```

Comment:
It would be nice if these checks could be done in one location ahead of calling this function, but I see that each case has different evaluations, so unless things were significantly refactored (out of scope for this defect), I don't think it would be feasible here.

Reply:
Agreed. I am considering condensing SZVAVModel.cc into one function and also making this type of change.